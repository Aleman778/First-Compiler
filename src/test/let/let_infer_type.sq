
// Type inference for let bindings is not supported currently by the compiler

fn main() {
    let a = 5;
}
