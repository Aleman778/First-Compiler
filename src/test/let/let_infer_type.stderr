error: mismatched types
 --> src/test/let/let_infer_type.sq:5:13
  |
5 |     let a = 5;
  |             ^ expected `()`, found `i32`

